pom.xml

```diff
@@ -840,6 +838,9 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.10.1</version>
+          <configuration>
+            <release>17</release>
+          </configuration>
```

Comment:
I think we should still keep the java 17 setting as propertie, like the other main project settings